concent_api/conductor/service.py

```diff
@@ -125,10 +125,10 @@ def _store_blender_crop_script_parameters(blender_parameters: Any) -> BlenderCro
         resolution_y=blender_parameters['resolution'][1],
         samples=blender_parameters['samples'],
         use_compositing=blender_parameters['use_compositing'],
-        borders_x_min=blender_parameters['borders_x'][0],
-        borders_x_max=blender_parameters['borders_x'][1],
-        borders_y_min=blender_parameters['borders_y'][0],
-        borders_y_max=blender_parameters['borders_y'][1],
+        borders_x_min=str(blender_parameters['borders_x'][0]),
+        borders_x_max=str(blender_parameters['borders_x'][1]),
+        borders_y_min=str(blender_parameters['borders_y'][0]),
+        borders_y_max=str(blender_parameters['borders_y'][1]),
```

Comment:
When fixing an error like this you should add a test for the failing case.  So `_store_blender_crop_script_parameters()` with the bad float values.

Also, a comment about why it's being converted to `str()` would be useful. It's not obvious.

Reply:
Yup, writing it right now. I've pushed it so @bartoszbetka could deploy it faster.

Reply:
Done